src/test/java/com/openshift/internal/restclient/model/build/WebhookTriggerTest.java

```diff
@@ -29,13 +29,13 @@ public void setup(){
 	}
 	@Test
 	public void testGetWebhookUrlWhenResourceHasBaseURL() {
-		trigger = new WebhookTrigger(BuildTriggerType.generic, "secret101","foo","https://localhost:8443","v1beta1","test");
-		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/generic?namespace=test", trigger.getWebhookURL());
+		trigger = new WebhookTrigger(BuildTriggerType.GENERIC, "secret101","foo","https://localhost:8443","v1beta1","test");
+		assertEquals("https://localhost:8443/osapi/v1beta1/buildConfigHooks/foo/secret101/Generic?namespace=test", trigger.getWebhookURL());
```

Comment:
Hmm...we do have some mixed cased URL values though since v1beta3, I believe case no longer matters.  Recall this enum is used in the API models where it is more significant then in the URL string.